mt7697/README.md

```diff
@@ -0,0 +1,18 @@
+# LINE Things Starter for Linkit 7697
+
+## Requirements
+* [Arduino IDE](https://www.arduino.cc/en/Main/Software)
+* [Linkit 7697](https://labs.mediatek.com/zh-tw/chipset/MT7697)
+* Micro-USB to USB Cable
+
+## Installation
+Please ensure you have Arduino IDE installed and the board **disconnected**.
+
+[Arduino IDE Setup](https://docs.labs.mediatek.com/resource/linkit7697-arduino/zh_tw/environment-setup/setup-arduino-ide)
+
+## Setup
+[Steps](https://docs.labs.mediatek.com/resource/linkit7697-arduino/zh_tw/environment-setup/connecting-linkit-7697-to-computer)
```

Comment:
I think correct URL is here.
https://docs.labs.mediatek.com/resource/linkit7697-arduino/zh_tw/environment-setup/setup-arduino-ide

```suggestion
[Steps](https://docs.labs.mediatek.com/resource/linkit7697-arduino/zh_tw/environment-setup/setup-arduino-ide)
```